src/components/Header.js

```diff
@@ -2,13 +2,13 @@ import "../style/header.css";
 import { Link } from "react-router-dom";
 import { useFilter } from "../context/filter-context";
 import { useState } from "react";
+import { useCart } from "../context/cart-context";
 
 function Header() {
-
+  const {state: {itemCount}} = useCart();
```

Comment:
You can destructure and pass the value from context itself

Reply:
ok